Neos.Neos/Resources/Private/Fusion/Prototypes/BreadcrumbMenu.fusion

```diff
@@ -1,17 +1,29 @@
 # Neos.Neos:BreadcrumbMenu provides a breadcrumb navigation based on menu items.
 #
-prototype(Neos.Neos:BreadcrumbMenu) < prototype(Neos.Neos:Menu) {
-  templatePath = 'resource://Neos.Neos/Private/Templates/FusionObjects/BreadcrumbMenu.html'
+prototype(Neos.Neos:BreadcrumbMenu) < prototype(Neos.Fusion:Component) {
+  attributes = Neos.Fusion:DataStructure
 
-  itemCollection = ${q(documentNode).parents('[instanceof Neos.Neos:Document]').get()}
-  // Show always the current node, event when it is hidden in index
-  items.@process.addCurrent = Neos.Fusion:Value {
-      currentItem = Neos.Neos:MenuItems {
-        renderHiddenInIndex = true
-        itemCollection = ${[documentNode]}
-      }
-      value = ${Array.concat(this.currentItem, value)}
+  @private {
+    items = Neos.Neos:BreadcrumbMenuItems {
+      @ignoreProperties = ${['attributes']}
+      @apply.props = ${props}
+    }
   }
 
-  attributes.class = 'breadcrumb'
+  renderer = Neos.Neos:MenuItemListRenderer {
+    items = ${private.items}
+    attributes = ${props.attributes}
+  }
+
+  @exceptionHandler = 'Neos\\Fusion\\Core\\ExceptionHandlers\\ContextDependentHandler'
+
+  @cache {
+    mode = 'cached'
+    entryIdentifier {
+      documentNode = ${Neos.Caching.entryIdentifierForNode(documentNode)}
+    }
+    entryTags {
+      1 = ${Neos.Caching.nodeTypeTag('Neos.Neos:Document', documentNode)}
```

Comment:
Could we only include the document nodes parents for flushing?

Reply:
We already flush for all documents

Reply:
So this way the DimensionsMenu and the Breadcrumb of every site is flushed when any document is changed in the website...

So IMO just including parents for both cases would solve both and reduce the flushing by a lot

Reply:
Decided not to do this as this pr is about de fluidizing not optimizing. Before the prototype inherited the cache annotations of menu.

There are plans to optimize the menus further so this is only the first step. This pr should not make anything worse but as it is but also should not need to fix more than it promises.

Would like to get this merged to look into the optimizing topic without piling too many changes upon each other.